examples/model_compression/distill_lstm/data.py

```diff
@@ -98,31 +92,43 @@ def _data_augmentation(data, whole_word_mask=whole_word_mask):
     np.random.seed(seed)
     used_texts, new_data = [], []
     for data in train_dataset:
-        data_list = tokenizer.tokenize(data[0])
-        used_texts.append(" ".join(data_list))
         if task_name == 'qqp':
-            data_list_2 = tokenizer.tokenize(data[1])
+            data_list = tokenizer.tokenize(data['sentence1'])
+            data_list_2 = tokenizer.tokenize(data['sentence2'])
+            new_data.append({
+                "sentence1": data_list,
+                "sentence2": data_list_2,
+                "labels": data['labels']
+            })
             used_texts.append(" ".join(data_list_2))
-            new_data.append([data_list, data_list_2, data[2]])
         else:
-            new_data.append([data_list, data[1]])
+            data_list = tokenizer.tokenize(data['sentence'])
+            new_data.append({"sentence": data_list, "labels": data['labels']})
 
+        used_texts.append(" ".join(data_list))
     used_texts = set(used_texts)
 
     for data in train_dataset:
         for _ in range(n_iter):
-            words, new_text = _data_augmentation(data[0])
             if task_name == 'qqp':
-                words_2, new_text_2 = _data_augmentation(data[1])
+                words, new_text = _data_augmentation(data['sentence1'])
+                words_2, new_text_2 = _data_augmentation(data['sentence2'])
                 if new_text not in used_texts or new_text_2 not in used_texts:
-                    new_data.append([words, words_2, data[2]])
-                    used_texts.add(new_text)
+                    new_data.append({
+                        "sentence1": words,
+                        "sentence2": words_2,
+                        "labels": data['labels']
+                    })
                     used_texts.add(new_text_2)
             else:
+                words, new_text = _data_augmentation(data['sentence'])
                 if new_text not in used_texts:
-                    new_data.append([words, data[1]])
-                    used_texts.add(new_text)
-    train_dataset.data.data = new_data
+                    new_data.append({
+                        "sentence": words,
+                        "labels": data['labels']
+                    })
+            used_texts.add(new_text)
+    train_dataset.new_data = new_data
```

Comment:
@smallv0221 Review the preprocessing pipeline here. Is it possible not to expose `new_data` variable in this example? @LiuChiachi